src/events.ts

```diff
@@ -37,6 +37,7 @@ export interface CommonAccessCodeEvent<
     EventType,
     {
       access_code_id: string
+      access_code_code: string
```

Comment:
I think it's ok to go with the more natural `code` here. The prefix is added always for ids and whenever there is ambiguity between a "joined" or "column common to more than one resource". e.g. we would clarify `device_name` or `access_code_name` if they were both involved in the event, but in this case `code` is unambiguous and more closely matches the underlying resource structure, e.g. we don't do `device_battery_level` on the device event.

```suggestion
      code: string
```